certif/app/styles/components/register-form.scss

```
@@ -1,32 +1,19 @@
 .register-form {
-  max-width: 360px;
-  overflow-y: auto;
-  font-size: 0.875rem;
+  display: flex;
+  flex-direction: column;
+  gap: $pix-spacing-m;
+  max-width: 450px;
 
-  &__cgu-label {
-    margin: 0;
-    font-family: $font-roboto;
+  &__information {
+    margin-bottom: $pix-spacing-s;
+    color: var(--pix-neutral-900);
+    text-align: center;
+
+    @extend %pix-body-s;
   }
 
-  &__cgu-error {
-    margin-top: 4px;
+  &__cgu-message--error {
     color: var(--pix-error-700);
     font-size: 0.75rem;
   }
```

Suggestion si on veut garder le modifier : 
```suggestion
  &__cgu-message {
     @extend %pix-body-xs; // pour utiliser le placeholder côté typographie
     color: currentColor; // optionnel mais c'est pour marquer le contraste avec le message avec / sans couleur
  
   &--error {
    color: var(--pix-error-700);
  }
```

---

vu plus haut, un oubli d'une tentative de retouche de fichier, j'enlève le modifier !